src/components/dashboard/forms/web3/WithdrawFromIdentity.js

```diff
@@ -33,9 +36,12 @@ class WithdrawFromIdentity extends Component {
 		this.props.validate('withdrawAmount', { isValid: isValid, err: { msg: msg, args: errMsgArgs }, dirty: dirty })
 	}
 
-	validateAddress = (addr, dirty) => {
-		const isValid = isEthAddress(addr)
-		const msg = 'ERR_INVALID_ETH_ADDRESS'
+	validateAddress = async (addr, dirty) => {
+		// As we are using async validation so we need to make 
+		// the form show a message that it is waiting for validation
+		this.props.validate('withdrawTo', { err: { msg: "ERR_VALIDATION_ASYNC" }, dirty: dirty });
```

Comment:
I'm not sure this is the right way to do it. Better use `updateSpinner('check-addr-${addr}')` action with `memory.spinners['check-addr-${addr}']`...

Reply:
I also thought about spinners. Will check it to see how I can do it.